apps/utils/lib/utils.ex

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Utils do
```

Comment:
General comments  on `Utils`:
1. do we really want such a generic app? I'd fear that it will end up being the "landfill" where all the code is going to end up in. Isn't there a more specific spot in the hierarchy we could put the functionality in here to?

    In fact - wouldn't we want `OMG.RPC` to be that spot? what's the separation of concerns between those two?
2. the naming: I see you anticipate dropping of the `OMG.` prefix, but we're kinda still discussing such change. Risking advocating a lost cause, I'd prefer we stick to the conventions and call it `OMG.Utils` (or `OMG.sth else`, wrt point 1. above) and `omg_utils`. If there's a convention adopted, let's be consistent.

Reply:
1. I think having a utill app is generally a good idea, as it gives you the ability to separate applications but keeping the same behaviour.
Utils was introduced because :omg_watcher used modules and functions from omg_rpc, but it's not sharing any other of the behavior omg_rpc does. I

2. The naming 
- I can't figure out what the intended prefix in our apps is supposed to mean (maybe because module naming is global?)
- having an application like we're having (just OMG) is problematic in general, because search becomes broken :D. I think what we would want is to know from the prefix, to which application a module belongs.
- I actually didn't do this on purpose I think. I forgot! :)